Library/Formula/dex.rb

```diff
@@ -18,7 +18,8 @@ class Dex < Formula
   def install
     args = ["prefix=#{prefix}",
             "CC=#{ENV.cc}",
-            "HOST_CC=#{ENV.cc}"]
+            "HOST_CC=#{ENV.cc}",
```

Comment:
Just an as FYI, rather than adding commas to end of arrays like this which looks a bit off, I personally prefer the:

```ruby
args = %W[
  prefix=#{prefix}
  CC=#{ENV.cc}
  HOST_CC=#{ENV.cc}
]
```
Style arrays. Up to you of course whether you follow that.